### PR TITLE
QUDV Icon adapted

### PR DESCRIPTION
**Pull request by fmbouga**
The purpose of this issue is to somehow change the current QUDV Icon shown on creating for example a new quantity, etc. with a better Icon. 

Extra changes related to author issues has been made.

### CODE REVIEW

File: de.dlr.sc.virsat.qudv.ui/resources/facets-page-wizban.png
Comment (PhilMFischer) [inline de.dlr.sc.virsat.qudv.ui/resources/facets-page-wizban.png:1]: Hi @fmbouga I think we should have an icon in the style we usually have in virtual satellite. Even though it is a nice icon i think it does not tell anything about QUDV or units in general. Maybe we should brainstorm a bit together.

File: .mailmap
Comment (PhilMFischer) [inline .mailmap:1]: Looks like there is a merge conflict that still needs to be resolved.

File: known_authors.txt [hunk 1]
```diff
@@ -17,3 +17,4 @@ maur_pa
 dellerDLR
 pchrszon-dlr
 DimitriDiantos
+mbou_fr
```
Comment (PhilMFischer) [inline known_authors.txt:20]: your GitHub account name should be added here.

**Review by ghost — CHANGES_REQUESTED** [review]
Please have a look to the comments. we can also work on the icon next week.

**Comment by PhilMFischer** [comment]
Please also add a description telling us what this PR is about.

**Comment by fmbouga** [comment]
@PhilMFischer alright thanks. Yeah it is true the Icon was like trying to show some symbols, quantity and unit all together. Next week we could then either generate it together with Copilot or find another way. Thanks for the other comments related to Known authors, description

**Comment by PhilMFischer** [comment]
Decided to completely remove the icon from the editor, since it created issues when rendering.

**Review by ghost — APPROVED** [review]
Looks great now- good idea to remove the icon.

**Review by kain-ap — APPROVED** [review]

**Comment by PhilMFischer** [comment]
Closes #29

**Pull request merged**